src/parser/shared/modules/items/bfa/pvp/DreadGladiatorsInsignia.js

```diff
@@ -0,0 +1,75 @@
+import React from 'react';
+
+import SPELLS from 'common/SPELLS/index';
+import ITEMS from 'common/ITEMS/index';
+import { formatPercentage } from 'common/format';
+import { calculatePrimaryStat } from 'common/stats';
+
+import Analyzer from 'parser/core/Analyzer';
+import Abilities from 'parser/shared/modules/Abilities';
+
+const BASE_ITEM_LEVEL = 280;
+const BASE_PRIMARY_STAT_BUFF = 63;
```

Comment:
Is this value correct? It sounds very low.
https://www.wowhead.com/item=161676/dread-gladiators-insignia
https://www.wowdb.com/items/161676-dread-gladiators-insignia

Reply:
The armory value looks way too low aswell, at a quick glance it is likely same value as wowhead (100 ilvl -> 3xprimary stat sounds like the correct ballpark). IDK if the wowdb value is correct either, but it's atleast in the correct range (277/2 = 138.5). Where as both armory and wowhead are both way off.

Reply:
Both wowhead and wowdb agree on the number if you give it a realistic itemlevel
https://www.wowhead.com/item=161676/dread-gladiators-insignia&bonus=4982:1532
https://www.wowdb.com/spells/277182-taste-of-victory?itemLevel=340&itemSlot=12&playerLevel=120